loopout_test.go

```diff
@@ -555,6 +584,15 @@ func testPreimagePush(t *testing.T) {
 
 	// The client should then register for a spend of the HTLC and attempt
 	// to sweep it using the custom confirmation target.
+	ctx.AssertRegisterSpendNtfn(swap.htlc.PkScript)
+	ctx.AssertRegisterSpendNtfn(swap.htlc.PkScript)
```

Comment:
So with just one sweep we'd monitor the spend twice right? Plus confirmation? Would be much simpler to just monitor the spend of the primary sweep and notify the swaps in the batch.